Error C3646 (in windows environment, under which MVSC compiles)

Hello @rwardlow01 ,

Thank you for a project that has inspired me so much.

If the order of `#include "imgui_wrapper.hpp"` and `#include "imgui_opencv.hpp"` is written in reverse when calling the header file, the MVSC compiler will generate a **C3646** error in the windows environment.

```cpp
// Normal: 
#include "imgui_wrapper.hpp"
#include "imgui_opencv.hpp"

// Error:  imgui_opencv.hpp(41,12): error C3646: “image_texture_”: 未知重写说明符 
#include "imgui_opencv.hpp"
#include "imgui_wrapper.hpp"
```

The reason for this is that the `imgui_opencv.hpp` file does not contain `imgui_wrapper.hpp` and the `GLuint` type is unknown. So I have made the following changes
```patch
diff --git a/FlowCV_SDK/third-party/imgui_wrapper/imgui_opencv.hpp b/FlowCV_SDK/third-party/imgui_wrapper/imgui_opencv.hpp
index fae905e..1281c6b 100644
--- a/FlowCV_SDK/third-party/imgui_wrapper/imgui_opencv.hpp
+++ b/FlowCV_SDK/third-party/imgui_wrapper/imgui_opencv.hpp
@@ -38,7 +38,7 @@ class ImGuiOpenCvWindow
 
   private:
     ImGuiOpencvWindowData window_data_{};
-    GLuint image_texture_{};
+    unsigned int image_texture_{};
     cv::Mat frame_;
     bool keep_aspect_{};
     bool init_texture_once_{};
```

If you also approve of my patch, I can create a PR, thanks

--- a/.gitignore
+++ b/.gitignore
@@ -1,4 +1,4 @@
-?uild/
+?uild*/
 cmake-build*/
 FlowCV_SDK/third-party/opencv/
 .idea/
@@ -14,6 +14,7 @@ compile_commands.json
 CTestTestfile.cmake
 _deps
 .vs/
+.vscode/
 *.pdb
 *.vcxproj
 *.vcxproj.filters

